src/.bazelrc

```diff
@@ -11,3 +11,6 @@ build --crosstool_top=//cc_toolchain:toolchain
 # Automatically set the CPU environment based on the `--cpu` flag as per our
 # defined CPU environments
 build --auto_cpu_environment_group=//cc_toolchain:cpus
+#build --remote_executor=grpc://localhost:8980
```

Comment:
What does this do?

Reply:
Whoops that was from testing remote building, removed.